src/telemetry/Makefile

```diff
@@ -0,0 +1,20 @@
+export GOPATH=/tmp/go
```

Comment:
better to move all these files into telemtry repo, the idea is that people can build debian package in the repo and can use it independent of other projects.

Reply:
Having problem creating debian package for the sonic-telemetry application in its own repo.  Probably need more time to figure out the appropriate way to do it.